components/rfid-reader/PN532/setup_pn532.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+HOME_DIR="/home/pi"
+JUKEBOX_HOME_DIR="${HOME_DIR}/RPi-Jukebox-RFID"
```

Comment:
Is it possible to define these globally somewhere?

Reply:
Yes, see answer above. :)